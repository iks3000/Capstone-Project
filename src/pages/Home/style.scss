@import '../../assets/styles/variables.scss';

.home {
    background: url(../../assets/images/mainBackground.png) 0 0 no-repeat;
    height: 100vh;
    background-size: cover;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    position: relative;
    text-align: center;
    color: $white;
}