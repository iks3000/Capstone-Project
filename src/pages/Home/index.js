import PhotoBox from '../../components/PhotoBox';
import Button from '../../components/Button';
import { Link } from "react-router-dom";
import UserAvatar from '../../assets/images/users/UserAvatar.png';
import './style.scss';

const Home = () => {
    return (
        <section className="home">
            <PhotoBox
                avatar={UserAvatar}
                name={'John Doe'}
                title={'Programmer. Creative. Innovator'}
                description={'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque.'}
            />
            <Link to='/inner'>
                <Button text={'Know more'} />
            </Link>
        </section>
    )
}

export default Home;