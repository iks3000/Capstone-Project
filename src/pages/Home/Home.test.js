import { shallow } from "enzyme";
import Home from ".";
it("should render Inner component", () => {
  expect(shallow(<Home />)).toMatchSnapshot();
});
