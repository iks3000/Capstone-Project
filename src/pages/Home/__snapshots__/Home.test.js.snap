// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Inner component 1`] = `
<section
  className="home"
>
  <PhotoBox
    avatar="UserAvatar.png"
    description="Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque."
    name="John Doe"
    title="Programmer. Creative. Innovator"
  />
  <Link
    to="/inner"
  >
    <Button
      text="Know more"
    />
  </Link>
</section>
`;
